### ISSUE TITLE
[Enhancement]: Ability to download entire Author and/or Series

### ISSUE THREAD

**Issue by pjones112000** [advplyr/audiobookshelf#3541]
### Type of Enhancement

Web Interface/Frontend

### Describe the Feature/Enhancement

Sometimes when I go on cruises I like taking audiobooks with me to listen offline...it would be nice to be able to download all the audiobooks for a specific or series rather than downloading one at a time....particularly on long cruises or vacations.

### Why would this be helpful?

Currently, I would either need to download one book at a time or copy directly from the filestore to my tablet/phone to listen rather than selecting an entire series or author to download all to my tablet/phone in an automated fashion.

### Future Implementation (Screenshot)

Add option to either the picture of the author or the author details page to download.

### Audiobookshelf Server Version

v2.14.0

**Comment by nichwall** [comment]
Do you mean in the web client or in the mobile app?

If the mobile app, duplicate of https://github.com/advplyr/audiobookshelf-app/issues/188

**Comment by pjones112000** [comment]
Sorry, I meant to mention, the web UI

**Comment by advplyr** [comment]
To further clarify, are you downloading the zip file on the web client and playing this in an audio player on your phone?

**Comment by pjones112000** [comment]
Yes, exactly.   Zip to phone, using an offline music player.  Admittedly,
an offline version of the webclient would be nice but I know that's more
complicated.

**Comment by pjones112000** [comment]
...or even an offline version of the mobile app.

**Comment by advplyr** [comment]
The mobile apps for android and ios support downloading for offline

**Comment by Chriss78** [comment]
Hello just adding/comment this to the Enhancment. A DL option for the WebUI would be helpful for eBook Libraries. Typically a User  DL a  .epub to the PC and copy it to their desired eReader eg Tolino. Especially for series with multiple parts can this be usful and spare a user much clicking each book and afterwards unzipping again etc.
Maybe in combine the funktion with the mark (circle in book cover top left corner) and DL the marked eBooks.
